.github/workflows/regression-revocation.yml

```diff
@@ -0,0 +1,89 @@
+name: Revocation
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      # See matrix of secrets...
+      # https://sbulav.github.io/terraform/github-actions-matrix-secrets/
+      matrix:
+        include:
+          - organization_did_web: TRANSMUTE_STAGING_ORGANIZATION_DID_WEB
+            client_id: TRANSMUTE_STAGING_CLIENT_ID
+            client_secret: TRANSMUTE_STAGING_CLIENT_SECRET
+            token_audience: TRANSMUTE_STAGING_TOKEN_AUDIENCE
+            token_endpoint: TRANSMUTE_STAGING_TOKEN_ENDPOINT
+            api_base_url: TRANSMUTE_STAGING_API_BASE_URL
+
+    steps:
+      - name: Begin CI...
+        uses: actions/checkout@v2
+      - name: Use Node 14
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+
+      - name: Run Tests
+        env:
+          organization_did_web: ${{secrets[matrix.organization_did_web]}}
+          client_id: ${{secrets[matrix.client_id]}}
+          client_secret: ${{secrets[matrix.client_secret]}}
+          token_audience: ${{secrets[matrix.token_audience]}}
+          token_endpoint: ${{secrets[matrix.token_endpoint]}}
+          api_base_url: ${{secrets[matrix.api_base_url]}}
+        run: |
+          npx newman run ./docs/postman/regression/revocation.collection.json \
+          --env-var ORGANIZATION_DID_WEB=$organization_did_web \
+          --env-var CLIENT_ID=$client_id \
+          --env-var CLIENT_SECRET=$client_secret \
+          --env-var TOKEN_AUDIENCE=$token_audience \
+          --env-var TOKEN_ENDPOINT=$token_endpoint \
+          --env-var API_BASE_URL=$api_base_url \
+          --reporters cli,jsonname: Revocation Tests
+
+on:
```

Comment:
this seems wrong?
probably should either be a job under the existing workflow OR in its own workflow file.

Reply:
This is in its own workflow file.

I think we should avoid putting all the jobs in the same file, especially for smaller set of basic feature validation.

It's nice to be able to confirm a simple feature (like revocation) in isolation form a wider range of tests.

I think ultimately we will need both small atomic tests, and larger integration tests.

Reply:
yes, I think orie has the right of it - basically quick feature tests that can be run directly, but also get incorporated in as part of baseline capabilities testing, and as a larger part of the interop suite 

@brownoxford - thoughts? basically I think modular is good

Reply:
I agree it should be on its own workflow, but here it looks like you have two separate workflows in one file, one to eventually run: ` npx newman run ./docs/postman/regression/revocation.collection.json` and one to eventually run `npx newman run ./docs/tutorials/revocation/revocation.collection.json `
not sure why the second one isnt just another step under the initial workflow or just another job under it.

Reply:
hmm, maybe i got the path wrong?

Reply:
I think I got the paths right, but if not my intention was:

1. a folder per collection with readme instructions on how to run it, and how it was built and screenshots
2. a workflow file per collection that allows a vendor to test that collection in isolation

I think I sorta accomplished these goals, but its possible I was tired when I made the PR and the paths are wrong.


Its a bit anoying since you can't test a PR like this easily until after it is merged, due to the use of `on dispatch`.

Reply:
I think what im trying to say is that you're testing the same workflow twice.
currently the file reads like this:

```
Name: Revocation
on: workflow_dispatch:
jobs: DO FIRST JOB
on: workflow_dispatch:
jobs: DO SECOND JOB
```
and both jobs look identical, except for that path, so either you made a mistake and are running the same job twice or they are indeed two distinct paths you want to test and in that case the second job's last step can just be appended to the first job.
if its still not clear happy to do the changes and show you what i mean

Reply:
ahh yes.... the path can probably be factored into a single workflow... I just didn't do that in this PR.

There are cases where we might need different env vars, but I think we should strive to eliminate that :)

happy to see this sorted out, in subsequent PR.

Reply:
> yes, I think orie has the right of it - basically quick feature tests that can be run directly, but also get incorporated in as part of baseline capabilities testing, and as a larger part of the interop suite
> 
> @brownoxford - thoughts? basically I think modular is good

I believe in separate units wherever possible, and we can aggregate them and pass data via environmental outputs.

Reply:
There is another issue to track this.